Classes/Service/WorkspaceActivityService.php

```diff
@@ -78,7 +78,12 @@ public function nodeDiscarded(NodeInterface $node): void
 
     public function shutdownObject(): void
     {
-        $currentUser = $this->securityContext->getAccount()->getAccountIdentifier();
```

Comment:
I would not skip the update of the workspace in general but instead skip updating the last editor and still update the last edited date time.